### PR TITLE
initial translation for `Window.queryLocalFonts()`

### PR DESCRIPTION
**Pull request by skyclouds2001**
### Additional details

https://github.com/mdn/content/blob/main/files/en-us/web/api/window/querylocalfonts/index.md?plain=1

https://developer.mozilla.org/en-US/docs/Web/API/Window/queryLocalFonts

### CODE REVIEW

File: files/zh-cn/web/api/window/querylocalfonts/index.md [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+### 返回值
+
+一个兑现一个表示本地可用字体的 {{domxref("FontData")}} 对象数组的 {{jsxref("Promise")}}。
+
+### 异常
+
+- `NotAllowedError` {{domxref("DOMException")}}
+  - : 首次调用该方法后，当浏览器出现权限提示时，用户选择拒绝授予此功能的权限。
+- `SecurityError` {{domxref("DOMException")}}
+  - : 此功能的使用被[权限策略](/zh-CN/docs/Web/HTTP/Permissions_Policy)阻止，或者不是缘于用户交互（例如按下按钮）调用或当前的 {{glossary("origin")}} 是一个不透明的来源。
```
Comment (jasonren0403) [inline files/zh-cn/web/api/window/querylocalfonts/index.md:36]:
```suggestion
  - : 此特性的使用被[权限策略](/zh-CN/docs/Web/HTTP/Permissions_Policy)阻止，或者不是通过用户交互（如按下按钮）调用此特性，或当前的{{glossary("origin", "源")}}不透明。
```

File: files/zh-cn/web/api/window/querylocalfonts/index.md [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+{{domxref("FontData.blob", "blob()")}} 方法提供对底层 [SFNT](https://zh.wikipedia.org/wiki/SFNT) 数据的访问——这是一种可以包含其他字体格式的字体文件格式，例如 PostScript、TrueType、OpenType 或 Web 开放字体格式（WOFF）。
+
+```js
+async function computeOutlineFormat() {
+  try {
+    const availableFonts = await window.queryLocalFonts({
+      postscriptNames: ["ComicSansMS"],
+    });
+    for (const fontData of availableFonts) {
+      // `blob()` 方法返回一个包含有效且完整的 SFNT 包装字体数据的 Blob。
```
Comment (jasonren0403) [inline files/zh-cn/web/api/window/querylocalfonts/index.md:87]:
```suggestion
      // `blob()` 方法返回一个包含有效且完整的 SFNT 封装字体数据的 Blob。
```

File: files/zh-cn/web/api/window/querylocalfonts/index.md [hunk 3]
```diff
@@ -0,0 +1,124 @@
...
+        case "\x00\x01\x00\x00":
+        case "true":
+        case "typ1":
+          outlineFormat = "truetype";
+          break;
+        case "OTTO":
+          outlineFormat = "cff";
+          break;
+      }
+      console.log("Outline format:", outlineFormat);
```
Comment (jasonren0403) [inline files/zh-cn/web/api/window/querylocalfonts/index.md:104]: “outline format” may be localized, but I'm not sure how 'outline' should be translated here.

/cc @yin1999

**Review by jasonren0403 — COMMENTED** [review]
some suggestions

File: files/zh-cn/web/api/window/querylocalfonts/index.md (hunk 3 above)
Comment (yin1999) [reply files/zh-cn/web/api/window/querylocalfonts/index.md:104]: https://zh.wikipedia.org/zh-cn/%E7%9F%A2%E9%87%8F%E5%AD%97%E4%BD%93

“矢量字体格式”

File: files/zh-cn/web/api/window/querylocalfonts/index.md [hunk 4]
```diff
@@ -0,0 +1,124 @@
...
```
Comment (skyclouds2001) [inline files/zh-cn/web/api/window/querylocalfonts/index.md:104]:
```suggestion
      console.log("矢量字体格式：", outlineFormat);
```

**Review by jasonren0403 — APPROVED** [review]
Thanks @skyclouds2001 and @yin1999 👍

**Pull request merged**